routes/mail/index.js

```diff
@@ -1,32 +1,28 @@
-"use strict";
+import request from 'superagent';
+const slackPostAPI = process.env.KINDLIZED_SLACK;
 
-var request = require('superagent');
-var log = require('common/log');
-var slackPostAPI = require('common/makeCredential')('slack');
+export default function(req, res) {
+  const mandrillEvents = req.body.mandrill_events;
+  let mandrillBody;
+  let mandrillHtml;
```

Comment:
__[jshint]__
*  'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).